amoro-shade-zookeeper-3/src/main/resources/META-INF/NOTICE

```diff
@@ -1,8 +1,5 @@
 amoro-shade-zookeeper-3
-Copyright 2014-2023 The Apache Software Foundation
```

Comment:
this copyright should be based on the copyright in the zookeeper notice

I actually think we should keep the notice from the zookeper jar and not replace it with an amoro one